crates/test-utils/src/lib.rs

```diff
@@ -1,46 +1,54 @@
-use std::sync::atomic::{AtomicUsize, Ordering};
-
 use arbitrary::{Arbitrary, Unstructured};
-use rand::{rngs::OsRng, RngCore};
+use rand::{rngs::OsRng, CryptoRng, RngCore};
 
 pub mod bitcoin;
 pub mod bridge;
 pub mod evm_ee;
 pub mod l2;
 
-const ARB_GEN_LEN: usize = 1 << 24; // 16 MiB
+// Smaller buffer size as compared to 2^24
+const ARB_GEN_LEN: usize = 128;
```

Comment:
Upgrade this comment to a full blown docstring.

Reply:
Maybe bump it up to 1024 at least?